packages/widgetbook_core/lib/src/themes/themes.dart

```diff
@@ -61,7 +61,7 @@ class Themes {
     outline: Color(0xFF8D9199),
     onInverseSurface: Color(0xFF1A1C1E),
     inverseSurface: Color(0xFFE3E2E6),
-    inversePrimary: Color(0xFF0060A7),
+    inversePrimary: Color(0xFFA1C9FF),
```

Comment:
Why is this changed?

Reply:
This was also a color that didn't match the design. I had planned to change it while I'm working on it and then revert it so I can revert it now.

Here's the color when changed (matches the design):

<img width="500" alt="image" src="https://user-images.githubusercontent.com/50345358/210273459-e71d9520-13e6-4379-aace-0b8a004f7d63.png">

<img width="400" alt="image" src="https://user-images.githubusercontent.com/50345358/210273618-d161a38b-477f-4120-9e65-363cd532f340.png">


And here is the color that was before (it's very blue)
<img width="500" alt="image" src="https://user-images.githubusercontent.com/50345358/210273316-d86e6247-ca93-4a42-810f-8b4403ca7571.png">

But I will revert it anyway because it might make sense to update the colors to match the design later in a separate PR.